models/glacier_exchange/avalanche_c/glacier_exchange_avalanche_c_schema.yml

```diff
@@ -0,0 +1,97 @@
+version: 2
+
+models:
+  - name: glacier_exchange_avalanche_c_trades
+    meta:
+      blockchain: avalanche_c 
+      sector: dex
+      project: glacier_exchange
+      contributors: discochuck 
+    config:
+      tags: ['avalanche_c','glacier_exchange','trades','dex']
+    description: >
+        glacier_exchange contract trades on avalanche_c 
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - blockchain
+            - project
+            - version
+            - tx_hash
+            - evt_index
+            - trace_address
+      - check_dex_seed:
+          blockchain: avalanche_c 
+          project: glacier_exchange
+          version: 1
```

Comment:
Hi @discochuck,
in order for `check_dex_seed` to work, can you add some seeds in this file?
https://github.com/duneanalytics/spellbook/blob/main/seeds/dex/trades/dex_trades_seed.csv